### ISSUE TITLE
How to import multiple Bruker 1D spectra

### ISSUE THREAD

**Issue by Ricmelis80** [NMRPy/nmrpy#3]
Hi, I triead reading and processing multiple raw 1D Bruker NMR spectra with NMRpy.  Bruker fids are orinally located in specific subfolders  es "1" (i.e the original Topspin EXPNO number) ,  For more clarity I have attached a zip files containing the data structure of the original root folder [nmr.zip](https://github.com/NMRPy/nmrpy/files/12586844/nmr.zip).

How I can process and display them? Thank you in advance!

**Comment by Ricmelis80** [comment]
Hi Johann, 

Thanks a lot for useful suggestions! I'll try to implement it

**Comment by jmrohwer** [comment]
Can this issue be closed?

**Comment by AdriennR** [comment]
Hi,
I kind of have the same issue, but my experiment folders with different numbers are in the same root folder as they have been generated by Topspin. When I try to import all the fid data into one FidArray, I receive the error message: "fid_path does not specify a valid .fid directory." I only can import fid data if the root folder contains one experiment folder only. 
Any suggestions?

**Comment by Ricmelis80** [comment]
Hy,

Sorry, I'm totally forget to close this iussue. I have implemented the author's suggestion and it works well!

**Comment by AdriennR** [comment]
Hi,
Interesting. It does not work for me though. :(

**Comment by jmrohwer** [comment]
> Hi, I kind of have the same issue, but my experiment folders with different numbers are in the same root folder as they have been generated by Topspin. When I try to import all the fid data into one FidArray, I receive the error message: "fid_path does not specify a valid .fid directory." I only can import fid data if the root folder contains one experiment folder only. Any suggestions?

Can you post the directory tree structure of your files? In the nmrpy install directory (under site-packages) there is some example test Bruker data, the structure looks like this:
```bash
tests/test_data/bruker2
├── 1
│   ├── WS_FTP.LOG
│   ├── acqu
│   ├── acqum
│   ├── acqums
│   ├── acqus
│   ├── cpdprg2
│   ├── fid
│   ├── format.temp
│   ├── pdata
│   │   └── 1
│   │       ├── 1i
│   │       ├── 1r
│   │       ├── WS_FTP.LOG
│   │       ├── meta
│   │       ├── meta.ext
│   │       ├── outd
│   │       ├── proc
│   │       └── procs
│   ├── pulseprogram
│   └── scon
├── 2
│   ├── WS_FTP.LOG
│   ├── acqu
│   ├── acqum
│   ├── acqums
│   ├── acqus
│   ├── cpdprg2
│   ├── fid
│   ├── format.temp
│   ├── pdata
│   │   └── 1
│   │       ├── 1i
│   │       ├── 1r
│   │       ├── WS_FTP.LOG
│   │       ├── meta
│   │       ├── meta.ext
│   │       ├── outd
│   │       ├── proc
│   │       └── procs
│   ├── pulseprogram
│   └── scon
├── 3
│   ├── WS_FTP.LOG
│   ├── acqu
│   ├── acqum
│   ├── acqums
│   ├── acqus
│   ├── cpdprg2
│   ├── fid
│   ├── format.temp
│   ├── pdata
│   │   └── 1
│   │       ├── 1i
│   │       ├── 1r
│   │       ├── WS_FTP.LOG
│   │       ├── meta
│   │       ├── meta.ext
│   │       ├── outd
│   │       ├── proc
│   │       └── procs
│   ├── pulseprogram
│   └── scon
...
...
```
You would import that with `fid_array = nmrpy.from_path('tests/test_data/bruker2')` (obviously you have to specify this relative to the current directory). The individual experiment folders are 1, 2, 3, ... under "bruker2" which is the root folder.

**Comment by AdriennR** [comment]
Yes, I first tried with the bruker2 experimental files. If I did that, it told me that there are only 3 fid files that can be imported (instead of the 20 ones present in the bruker2 folder's subfolders). 
THen I also tried with my files, and it has written the error message OSError: Data could not be imported.. 
My command: fid_array = nmrpy.from_path('/Users/User/Desktop/doktori/Lyon-Strassbourg/NMR/DNP/Measuremens/20230228_LNP_C2')
